src/cpp/session/modules/SessionRCompletions.R

```diff
@@ -1767,6 +1768,20 @@ assign(x = ".rs.acCompletionTypes",
    # parameterized R Markdown documents
    if (.rs.injectKnitrParamsObject(documentId))
       on.exit(.rs.removeKnitrParamsObject(), add = TRUE)
+
+   # If custom completions have been set through
+   # 'rc.options("custom.completions")',
+   # then use the internal R completions instead.
+   if (.rs.isCustomCompletionsEnabled()) {
+      
+      completions <- tryCatch(
+         .rs.getCustomRCompletions(line),
+         error = identity
+      )
+      
+      if (!inherits(completions, "error"))
```

Comment:
Should this error be logged somewhere? I'm thinking of the case where someone is expecting custom completions but not getting them, and doesn't know whether (a) they aren't being registered correctly, or (b) they are registered but triggering errors when run.